safe-global/web-core
# add: data management tab with import all data button

## What it solves
For firefox and brave users the automated migration does not work. 

Resolves #1097 

## How this PR fixes it
- Adds new `Data` tab to `Settings` page
- contains one button to `Import All Data`
- Import dialog to upload json file with localStorage of old safe app
- shows a summary of parsed json data
- Overwrites local storage when 'Import' is clicked

## How to test it
- Export data from an old safe  (see https://github.com/safe-global/safe-react/pull/4114)
- Navigate to Settings / Data and import the exported data there

## Analytics changes
Events for new import all feature:
- Button click
- Import finished

## Screenshots
https://user-images.githubusercontent.com/2670790/200835380-3ead22ca-0a00-480a-aa53-ad69d6d4ad0b.mov
